docs/mobility/slam/cartographer/_index.md

```diff
@@ -78,36 +78,53 @@ See Viam's [Pricing](https://www.viam.com/product/pricing) for more information.
 To create a new map, follow the instructions below.
 Creating a new map uses an instance of the cartographer module running in the cloud.
 
-1. Configure your `cartographer` SLAM service
+1. Enable data capture and configure your `cartographer` SLAM service
 
-   After installing your physical RPlidar and adding the `rplidar` module as outlined in the [requirements](#requirements) section, follow the steps below to add the `cartographer` module to your machine:
+   Follow the steps below to enable data capture and add the `cartographer` module to your machine:
 
    {{< tabs name="Create new map">}}
    {{% tab name="Config Builder" %}}
-   Follow the instructions below to set up the `cartographer` module on your machine:
 
-   1. Navigate to the **Config** tab of your machine's page in [the Viam app](https://app.viam.com).
-   1. Click on the **Services** subtab and click **Create service** in the lower-left corner.
-   1. Select **SLAM**, then select `cartographer`.
+   Add the data management service:
+
+   1. From your machine's **Config** tab, navigate to the **Services** subtab.
+   2. Click **Create service** in the lower-left corner of the page.
+      Choose `Data Management` as the type and specify a name for your data management service, for example `Data-Management-Service`.
+   3. Click **Create**.
+   4. On the panel that appears, you can manage the capturing and syncing functions and specify the **directory**, the sync **interval** and any **tags** to apply to captured data. See the [data management service](/data/) for more information.
+
+   Enable data capture for your camera, and for your movement sensor if you would like to use a movement sensor to provide IMU input:
+
+   5. Find the component's card on your machine's **Config** tab.
+   6. Click `Add Method` and then select the method type and the capture frequency. For the camera, choose the [`NextPointCloud`](/components/camera/#getpointcloud) method. For a movement sensor, choose the [`AngularVelocity`](/components/movement-sensor/#getangularvelocity) and [`LinearAcceleration`](/components/movement-sensor/#getlinearacceleration) methods.
+      We recommend a capture frequency of `5` Hz for RPlidar cameras and `20` Hz for movement sensors.
+   
+   {{< alert title="Tip" color="tip" >}}
```

Comment:
Thanks for moving this up here! This alert also currently appears under "Update an existing map", but I think we could remove it from there and just have it once here

Reply:
I also found an alert in the "Localize Only" section and removed, is that ok or would you want to keep that one in? @EshaMaharishi

Reply:
Oh, good catch, not sure why that alert was there under "Localize Only" since data capture isn't even used there. Sounds good to remove!